source/user-manual/capabilities/vulnerability-detection/known-issues.rst

```diff
@@ -0,0 +1,90 @@
+
+Known issues
+============
+
+This section covers known issues that might occur when using the Vulnerability Detection module.
+It also describes the version that contains the fix, and the workaround to apply when the upgrade isn't possible.
+
+Dangling entries found in the vulnerabilities dashboard
+-------------------------------------------------------
+
+.. list-table:: Issue description
+   :widths: 15 50
+
+   * - Problem
+     - There are vulnerabilities reported from agents that were removed from the environment. This issue also presents as vulnerabilities reported about packages that are no longer installed (removed or upgraded).
+   * - Affected versions
+     - Wazuh 4.8.0 and later. Before this version, the vulnerabilities weren't indexed.
+   * - Cause
+     - The internal mechanism that handled the agent removal event misses the event and it isn't processed. This can happen for example when there are queued scan events and a content update triggers a re-scan.
+   * - Fix available
+     - Wazuh 4.10.0 has a fix that prevents this from happening again. Now an agent removal event won't be queued but be processed right away. This avoids the dangling entries in the vulnerabilities dashboard (https://github.com/wazuh/wazuh/pull/26232). Nevertheless, the upgrade to this version won't remove the prior dangling entries, so the workaround should be applied manually.
+
+Known configuration/environment issues that cause a similar behavior
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To confirm whether you are affected by this issue, first, some common causes must be discarded.
+
+- If the problem is related to packages that are no longer installed, it's required to confirm the agent is active and reporting. Verify also in the agent's logs that there aren't error/warning messages related to the connection with the manager. Make sure the package isn't reported in the agents' inventory (if it is, there might be a problem with `syscollector` instead).
+- The manager is in charge of updating the vulnerabilities inventory. Confirm the manager's logs don't show any error/warning messages related to the connection with Wazuh Indexer, for example: `No available server` or `Failed to sync agent 'X' with the indexer`.
+- The Indexer cluster status is healthy. You can query this like `GET _cluster/health`, and it should be 'green' for Wazuh v4.8.0-v4.9.0 and 'green' or 'yellow' for Wazuh v4.9.1 and later (https://github.com/wazuh/wazuh/pull/25417).
+- Some time has passed since the agent was removed or the package was upgraded. When an agent updates its inventory, it's required to wait for the next :doc:`Syscollector </user-manual/capabilities/system-inventory/configuration>` scan before the changes are reported.
+
+First, correct any of the situations described above before proceeding with the workaround.
+
+Workaround
```

Comment:
```suggestion
First, correct any of the situations described above before proceeding with the cleanup.

Cleanup
```

Reply:
As we talked, an alternative is proposed.